api/__tests__/events.test.js

```diff
@@ -38,19 +38,77 @@ const scope = nock('http://localhost:5000')
         "createdTime": "2021-09-30T17:41:59.000Z"
       }])
 
+      const scopeSingleEvent = nock('http://localhost:3000')
+      .get('/airtable/event/recAb4kAFdkeKKnLU')
+      .reply(200, 
+        {
+          "id": "recAb4kAFdkeKKnLU",
+          "fields": {
+            "Event Name": "Showcase - Volunteer app",
```

Comment:
Maybe this isn't a priority at this stage or would be a pain as some of the AirTable column names contain spaces and capitals, but I wondered if we should use a consistent case in the event JSON object?  (not just in the test but I spotted it here) -- e.g. `id`, `createdTime`, etc are camel case but `Event Name`, etc aren't.  Don't want to hold things up so maybe not essential, but it's something that struck me seeing the test data.

Reply:
This is a copy of how Airtable returns the results.  I just copied and pasted the JSON from insomnia, it might need changed on the airtable itself